commandapi-core/src/main/java/dev/jorel/commandapi/AbstractCommandAPICommand.java

```diff
@@ -272,13 +273,18 @@ private void checkHasExecutors() {
 
 	@Override
 	public void register() {
+		register("minecraft");
+	}
+
+	@Override
+	public void register(@NotNull String namespace) {
```

Comment:
I don't tend to explicitly add `@Nonnull` annotations on parameters - our assumption is that everything that a user interacts with in the CommandAPI will be non-null _unless_ explicitly specified with a `@Nullable` annotation, suitable JavaDocs and something to bring a reader's attention in the documentation that it may be null. That way, if we ever get any null pointer exceptions, we can safely assume it's _always a user error_.

Otherwise, if you're going to add a not-null annotation explicitly, I'd expect a null pointer exception check as the first statement in this method.